### ISSUE TITLE
ktlint_disabled_rules does not work using editorConfigOverride

### ISSUE THREAD

**Issue by ihs7** [pinterest/ktlint#1675]
```
spotless {
    kotlin {
        ktlint('0.47.1').editorConfigOverride([
                disabled_rules: "ij_kotlin_allow_trailing_comma, ij_kotlin_allow_trailing_comma_on_call_site"
        ])
    }
}
```

This throws a lot of warnings `Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.`

When changing to:
```
spotless {
    kotlin {
        ktlint('0.47.1').editorConfigOverride([
                ktlint_disabled_rules: "ij_kotlin_allow_trailing_comma, ij_kotlin_allow_trailing_comma_on_call_site"
        ])
    }
}
```

I get:
```
> Task :spotlessKotlin FAILED
Step 'ktlint' found problem in '<filename>.kt':
Can not create an EditorConfigOverride without properties. Use 'emptyEditorConfigOverride' instead.
java.lang.IllegalArgumentException: Can not create an EditorConfigOverride without properties. Use 'emptyEditorConfigOverride' instead.
```

## Expected Behavior
I expect that `ktlint_disabled_rules` should work using `editorConfigOverride` or using `disabled_rules` does not create warnings.

## Current Behavior
`ktlint_disabled_rules` does not work using `editorConfigOverride` and `disabled_rules` creates a lot of deprecation warnings.

## Additional information
* Current version of ktlint: 0.47.1

**Comment by paul-dingemans** [comment]
This is already fixed in PR #1671 but not yet merged.

**Comment by ihs7** [comment]
Ah got it! Thanks. I'll close this issue.
